components/brave_welcome_ui/storage.ts

```diff
@@ -10,7 +10,8 @@ const keyName = 'welcome-data'
 export const defaultState = {
   searchProviders: [],
   browserProfiles: [],
-  hideSearchOnboarding: false
+  showSearchCard: true,
+  showRewardsCard: true
```

Comment:
Should this default to `false` to match the feature flag?

Reply:
Yes! fixed and pushed 😄